crates/wysiwyg/src/composer_state.rs

```diff
@@ -32,9 +32,9 @@ where
 {
     pub fn new() -> Self {
         Self {
-            dom: Dom::new(Vec::new()),
-            start: Location::from(0),
-            end: Location::from(0),
+            dom: Dom::default(),
+            start: Location::default(),
+            end: Location::default(),
```

Comment:
A `usize` defaults to 0 so I thought it made sense to use the default here.

I didn't replace any calls like `Location::from(0)` in tests because I thought that would detract from the clarity of the tests, whereas here `ComposerState::new()` is only ever called once, from base.rs, so thought it was (marginally) clearer that we're setting up the default state and default locations by writing it like this.